### DISCUSSION TITLE
Xiaomi Poco X3 Pro - Back 2 camera, real B/W?

### DISCUSSION THREAD

**Discussion by 28446** [KillerInk/FreeDcam#215] [Q&A]
Hi FreDcam dev/users,
I'm playing whit B/W camera on my smart phone, whit the one that FreeDCam detect as B/W camera, "**Back 2**".
Looking at the image produced by FreeDCam I've spotted some burned pixels, but those are colored, some R, some B and some G.

My question: Is there a real B/W sensor under "**Back 2**" lenses or B/N is achieved by a software post production?

Thanks 
Regards 

[B/W sample](https://i.postimg.cc/8PWtkwc7/photo-2023-06-29-11-33-55.jpg)

**Comment by KillerInk** [comment]
Hi its hard to say. 
but i dont think its a bw sensor. also that looks not like hot or dead pixels.
if zoom in into the image and look into the wall corner you see there also a color casting.
overall there is some blue and green color in the crossing between black and white.
for me it looks like its the 2mp depth sensor.
